.circleci/config.yml

```diff
@@ -2,30 +2,33 @@ version: 2
 jobs:
   build:
     docker:
-      - image: trussworks/circleci-docker-primary:d0dd0a134c842e234d1df641884a2b6036def4a4
+      - image: trussworks/circleci-docker-primary:93fe471597189fed29f1ab2f517fc4c3370f2a77
         environment:
           - TEST_RESULTS: /tmp/test-results
-    working_directory: ~/go/src/github.com/trussworks/truss-aws-tools
+          - GO111MODULE: auto
     steps:
       - checkout
       - restore_cache:
           keys:
             - dot-cache-pre-commit-{{ checksum ".pre-commit-config.yaml" }}
       - restore_cache:
           keys:
-            - v1-vendor-{{ checksum "Gopkg.lock" }}
-      - run: echo 'export PATH=${PATH}:~/go/bin' >> $BASH_ENV
-      - run: go get -u github.com/golang/dep/cmd/dep
+            - go-mod-sources-v1-{{ checksum "go.sum" }}
+      - run:
+          name: Adding go binaries to $PATH
+          command: |
+            echo 'export PATH=${PATH}:~/go/bin' >> $BASH_ENV
+            source $BASH_ENV
       - run: curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(go env GOPATH)/bin v1.17.1
-      - run: mkdir -p "${TEST_RESULTS}"
+      - run: go get github.com/jstemmer/go-junit-report
       - run: make all
       - save_cache:
           key: dot-cache-pre-commit-{{ checksum ".pre-commit-config.yaml" }}
           paths:
             - ~/.cache/pre-commit
       - save_cache:
-          key: v1-vendor-{{ checksum "Gopkg.lock" }}
+          key: go-mod-sources-v1-{{ checksum "go.sum" }}
           paths:
-            - ~/go/pkg/dep
+            - "/go/pkg/mod"
```

Comment:
is this no longer relative to the homedir? This step was was skipped in CircleCI so I don't know if it ran:

`Skipping cache generation, cache already exists for key: go-mod-sources-v1-EDKXJWUBBdlpntlKFlmDyoCvChJXdODK7Da0wDxR0uI=
Found one created at 2019-08-05 21:29:27 +0000 UTC`

Reply:
Ooh good catch!